File: specification/specs/modules/snapshot_and_restore/repositories/verify_repository/verify_repository_request.ts

/**namespace:Modules.SnapshotAndRestore.Repositories.VerifyRepository */
interface verify_repository_request extends request {
	/**ambiguous_origin*/
	MasterTimeout: time;
	/**ambiguous_origin*/
	Timeout: time;
	/**ambiguous_origin*/
	Source: string;
	/**ambiguous_origin*/
	FilterPath: string;
}

File: output/csharp/Modules/SnapshotAndRestore/Repositories/VerifyRepository/VerifyRepositoryRequest.cs
using Nest.CommonOptions;
using Nest.CommonAbstractions;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.Modules {

	public class VerifyRepositoryRequest : RequestBase {
		
		[DataMember(Name="master_timeout")]
		public Time MasterTimeout { get; set; }


		[DataMember(Name="timeout")]
		public Time Timeout { get; set; }

	}
}
